content/es/docs/concepts/storage/storage-capacity.md

```diff
@@ -0,0 +1,75 @@
+---
+reviewers:
+  - edithturn
+  - raelga
+  - electrocucaracha
+title: Capacidad de Almacenamiento
+content_type: concept
+weight: 45
+---
+
+<!-- overview -->
+
+La capacidad de almacenamiento es limitada y puede variar según el nodo en el que un Pod se ejecuta: es posible que no todos los nodos puedan acceder al almacenamiento conectado a la red o que, para empezar, el almacenamiento sea local en un nodo.
+
+{{< feature-state for_k8s_version="v1.19" state="alpha" >}}
```

Comment:
In the latest version of this document the feature-state for v1.19 was removed.

Reply:
> Ya veo!  La versión en Inglés también considera v1.19, deberiamos removerlos en la versión en español?

Reply:
@gamba47 do you have any suggestions about this? :)
Should I remove it?

Reply:
I would remove it.

Reply:
@gamba47 I saw it on the English version:

"Storage capacity is limited and may vary depending on the node on which a pod runs: network-attached storage might not be accessible by all nodes, or storage is local to a node to begin with.

FEATURE STATE: Kubernetes v1.21 [beta] "

It is just v1.21, so I will remove it :)
Thank you!